devops/big-dipper/big-dipper.service

```diff
@@ -0,0 +1,20 @@
+[Unit]
+Description=big-dipper
+Requires=docker.service network-online.target 
+After=docker.service network-online.target
+
+[Service]
+TimeoutStartSec=5
+Restart=always
+User=root
+Group=root
+WorkingDirectory=/root/
+ExecStartPre=/usr/bin/docker-compose pull --quiet --ignore-pull-failures
+ExecStartPre=/usr/bin/docker-compose build --pull
+ExecStart=/usr/bin/docker-compose up --remove-orphans
```

Comment:
Never used docker-compose. Wonder where the target is set (i expect a repo reference or such)